src/decaylanguage/data/MintDalitzSpecialParticles.csv

```diff
@@ -7,20 +7,20 @@ ID,Mass,MassUpper,MassLower,Width,WidthUpper,WidthLower,I,G,P,C,Anti,Charge,Rank
 9991,9990,10,10,9900000000,30,30,0,1,-1,1,0,0,0,0,NonResP,??,0^{-}
 9993,9990,10,10,9900000000,30,30,1/2,1,-1,5,1,0,0,0,NonResV,??,1^{-}
 -9993,9990,10,10,9900000000,30,30,1/2,1,-1,5,1,0,0,0,NonResV,??,1^{+}
-998100,9990,10,10,9900000000,30,30,0,1,1,1,0,0,0,0,PiPi0,x(uU+dD)+y(sS),\left[\pi^{+}\pi^{-}\right]^{L=0}_{0}
-998101,9990,10,10,9900000000,30,30,0,1,1,1,0,0,0,0,PiPi1,x(uU+dD)+y(sS),\left[\pi^{+}\pi^{-}\right]^{L=0}_{1}
-998102,9990,10,10,9900000000,30,30,0,1,1,1,0,0,0,0,PiPi2,x(uU+dD)+y(sS),\left[\pi^{+}\pi^{-}\right]^{L=0}_{2}
-998103,9990,10,10,9900000000,30,30,0,1,1,1,0,0,0,0,PiPi3,x(uU+dD)+y(sS),\left[\pi^{+}\pi^{-}\right]^{L=0}_{3}
-998104,9990,10,10,9900000000,30,30,0,1,1,1,0,0,0,0,PiPi4,x(uU+dD)+y(sS),\left[\pi^{+}\pi^{-}\right]^{L=0}_{4}
-998105,9990,10,10,9900000000,30,30,0,1,1,1,0,0,0,0,PiPi5,x(uU+dD)+y(sS),\left[\pi^{+}\pi^{-}\right]^{L=0}_{5}
-998106,9990,10,10,9900000000,30,30,0,1,1,1,0,0,0,0,PiPi6,x(uU+dD)+y(sS),\left[\pi^{+}\pi^{-}\right]^{L=0}_{6}
-998110,9990,10,10,9900000000,30,30,0,1,1,1,0,0,0,0,KPi0,??,\left[K^{-}\pi^{+}\right]^{L=0}_{0}
-998111,9990,10,10,9900000000,30,30,0,1,1,1,0,0,0,0,KPi1,??,\left[K^{-}\pi^{+}\right]^{L=0}_{1}
-998112,9990,10,10,9900000000,30,30,0,1,1,1,0,0,0,0,KPi2,??,\left[K^{-}\pi^{+}\right]^{L=0}_{2}
-998113,9990,10,10,9900000000,30,30,0,1,1,1,0,0,0,0,KPi3,??,\left[K^{-}\pi^{+}\right]^{L=0}_{3}
-998114,9990,10,10,9900000000,30,30,0,1,1,1,0,0,0,0,KPi4,??,\left[K^{-}\pi^{+}\right]^{L=0}_{4}
-998115,9990,10,10,9900000000,30,30,0,1,1,1,0,0,0,0,KPi5,??,\left[K^{-}\pi^{+}\right]^{L=0}_{5}
-998116,9990,10,10,9900000000,30,30,0,1,1,1,0,0,0,0,KPi6,??,\left[K^{-}\pi^{+}\right]^{L=0}_{6}
+998101,9990,10,10,9900000000,30,30,0,1,1,1,0,0,0,0,PiPi0,x(uU+dD)+y(sS),\left[\pi^{+}\pi^{-}\right]^{L=0}_{0}
```

Comment:
There are no tests for these PDG IDs but I assume you prepared them according to the properties you need. So fine.